content/management/v1/core-resources/collaborators/add-collaborator.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Add a Collaborator
+---
+
+You can set some of the fields available in the collaborator object, below we only list the properties in the example and the possible required fields.
+
+| Property | Description |
+|---|---|
+| `collaborator` | The [collaborator object](#core-resources/collaborators/the-collaborator-object) |
+| `email` | Email that will be used in the space interface for collaborator or collaborator SSO ID **required** |
+| `role` | Role name of the collaborator, could be admin, editor or custom roles, which is set to `id` and set to `multi` if you have more than one role **required** |
```

Comment:
The part about the `id` doesn't sound clear to me. We should state clearly that in case of a custom role you need to send the role id.

Reply:
👍🏽